SupersonicSound/Studio/System.cs

```diff
@@ -68,9 +68,11 @@ private void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                }
+                    LowLevelSystem.Dispose();
+                    LowLevelSystem = null;
 
-                _system.release().Check();
+                    _system.release().Check();
```

Comment:
I *think* you actually want these done outside. That way the low level system gets disposed even if you forget to call dispose and the finalizer runs.